txn_writer.go

```diff
@@ -464,8 +464,6 @@ func diffLocks(l1 gps.Lock, l2 gps.Lock) *LockDiff {
 				diff.Add = append(diff.Add, add)
 				i2next = i2 + 1 // Don't evaluate to this again
 				continue        // Keep looking for a matching project
-			case +1: // Project has been removed, handled below
-				break
```

Comment:
This `break` is an innocent no-op. `staticcheck` looks for cases where the author mistakenly thought they were breaking out of an outer loop, but were really just breaking out of the `switch`. That seems like a good check to have. I think it's worth deleting this innocent code if it lets us incorporate `staticcheck` into CI.

We could also make `staticcheck` happy by doing this:
```go
case +1: // Project has been removed, handled below
    // no code here
```